### ISSUE TITLE
adapt-contrib-slider error

### ISSUE THREAD

**Issue by Deathjam** [adaptlearning/adapt_authoring#1537]
if you have a slider in the course,  answer the slider then close the course, once you go back in and try to resume, there should be other questions and text after the slider, but the entire page except for the top bar (with the back button and progress bar) is blank.

chrome dev tools error

```
adapt-contrib-slider.js:149 Uncaught TypeError: Cannot read property 'prop' of undefined
    at n.setAllItemsEnabled (adapt-contrib-slider.js:149)
    at n.disableQuestion (adapt-contrib-slider.js:130)
    at n.onEnabledChanged (questionView.js:46)
    at _ (backbone.js:2)
    at m (backbone.js:2)
    at f (backbone.js:2)
    at n.u.trigger (backbone.js:2)
    at n.set (backbone.js:2)
    at n.set (lockingModel.js:11)
    at n.setQuestionAsSubmitted (questionModel.js:131)
setAllItemsEnabled	@	adapt-contrib-slider.js:149
disableQuestion	@	adapt-contrib-slider.js:130
onEnabledChanged	@	questionView.js:46
_	@	backbone.js:2
m	@	backbone.js:2
f	@	backbone.js:2
u.trigger	@	backbone.js:2
set	@	backbone.js:2
set	@	lockingModel.js:11
setQuestionAsSubmitted	@	questionModel.js:131
setQuestionAsSubmitted	@	questionView.js:204
restoreUserAnswers	@	adapt-contrib-slider.js:121
setupQuestion	@	adapt-contrib-slider.js:30
preRender	@	questionView.js:35
initialize	@	adaptView.js:13
initialize	@	componentView.js:20
e.View	@	backbone.js:2
n	@	backbone.js:2
n	@	backbone.js:2
n	@	backbone.js:2
n	@	backbone.js:2
n	@	backbone.js:2
addChildren	@	adaptView.js:61
postRender	@	adaptView.js:20
(anonymous)	@	adaptView.js:34
(anonymous)	@	underscore.js:5
```

**Comment by tomgreenfield** [comment]
Hi @Deathjam, this issue appears to be a duplicate of https://github.com/adaptlearning/adapt_framework/issues/1394.

**TL;DR**: this was fixed in Slider v2.1.2 so it should just be a case of upgrading.

**Comment by Deathjam** [comment]
ah ok
